__tests__/spec/sanity-checks.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env jest */
+var request = require('supertest')
+var app = require('../../app.js')
+var path = require('path')
+var fs = require('fs')
+var assert = require('assert')
+
+/**
+ * Basic sanity checks on the dev server
+ */
+describe('The Prototype Kit', () => {
+  test('should generate assets into the /public folder', () => {
+    assert.doesNotThrow(function () {
+      fs.accessSync(path.resolve(__dirname, '../../public/javascripts/application.js'))
+      fs.accessSync(path.resolve(__dirname, '../../public/images/unbranded.ico'))
+      fs.accessSync(path.resolve(__dirname, '../../public/stylesheets/application.css'))
+    })
+  })
+
+  test('should send with a well formed response for the index page', async () => {
+    const response = await request(app).get('/')
+    expect(response.statusCode).toBe(200)
```

Comment:
Can we aim for one expectation per test?

Reply:
Good shout, this has now been addressed 👍